operator/pkg/reconciliation/constructor.go

```diff
@@ -404,7 +405,11 @@ func buildContainers(dc *api.CassandraDatacenter, serverVolumeMounts []corev1.Vo
 	// server logger container
 	loggerContainer := corev1.Container{}
 	loggerContainer.Name = "server-system-logger"
-	loggerContainer.Image = "busybox"
+	if baseImageOs := os.Getenv(api.EnvBaseImageOs); baseImageOs != "" {
+		loggerContainer.Image = baseImageOs
+	} else {
+		loggerContainer.Image = "busybox"
```

Comment:
not for this PR, but we should offer a way to shut off this tailing container

Reply:
Agreed